plugins/thunderbolt/fu-thunderbolt-device.c

```diff
@@ -171,11 +177,31 @@ fu_thunderbolt_device_check_safe_mode (FuThunderboltDevice *self)
 	fu_device_set_metadata_boolean (FU_DEVICE (self), FU_DEVICE_METADATA_TBT_IS_SAFE_MODE, TRUE);
 }
 
+static const gchar*
+fu_thunderbolt_device_type_to_string (FuThunderboltDevice *self)
+{
+	if (self->device_type == FU_THUNDERBOLT_DEVICE_TYPE_HOST_CONTROLLER) {
+		if (self->gen >= 4)
+			return "USB4 host controller";
```

Comment:
is it USB4 or `USB 4`? I've not been following all the branding stuff...

Reply:
`USB4™` https://www.usb.org/document-library/usb4tm-specification

Reply:
I don't think we want the TM else we'll have to scatter them all over the place :)